connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoReader.scala

```diff
@@ -81,11 +81,13 @@ private[kusto] object KustoReader {
         request.clientRequestProperties)
     }
 
+    KDSU.logInfo(myName, s"Finished exporting from kusto to '${storage.account}/${storage.container}/$directory'" +
+      s", will start parquet reading now")
+
     val path = s"wasbs://${storage.container}@${storage.account}.blob.core.windows.net/$directory"
     val rdd = try {
-      request.sparkSession.read.parquet(s"$path").rdd
-    }
-    catch {
+      request.sparkSession.read.parquet(s"$path").queryExecution.executedPlan.execute().asInstanceOf[RDD[Row]]
```

Comment:
can you explain what this does? or maybe link to some docs?

Reply:
not related to docs - took from another connector - instead of casting each element individually this casts the whole dataframe